.pre-commit-config.yaml

```
@@ -75,6 +75,20 @@ repos:
   hooks:
   - id: forbid-html-img-without-alt-text
 
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: 'v0.910-1'
+  hooks:
+  - id: mypy
+    additional_dependencies:
+    - typing
+    - types-six
+    args:
+    - --ignore-missing-imports
+    - --install-types
+    - --non-interactive
+    - cheroot/
+    pass_filenames: false
```

> Sorry, I don't get this point

I meant something like

```suggestion
- repo: https://github.com/pre-commit/mirrors-mypy
  rev: v0.910-1
  hooks:
  - id: mypy
    additional_dependencies:
    - typing
    - types-six
    args:
    - --ignore-missing-imports
    - --install-types
    - --non-interactive
    - --python-version
    - 3.5
    - cheroot/
    pass_filenames: false
- repo: https://github.com/pre-commit/mirrors-mypy
  rev: v0.910-1
  hooks:
  - id: mypy
    additional_dependencies:
    - typing
    - types-six
    args:
    - --ignore-missing-imports
    - --install-types
    - --non-interactive
    - --python-version
    - 2.7
    - cheroot/
    pass_filenames: false
```